concepts/key-concepts.md

```diff
@@ -62,74 +66,79 @@ streams.
 
 ## Filtering
 
-In some cases it is required to perform modifications on the Events content, the process to alter, enrich or drop Events is called Filtering.
+You might need to perform modifications on the Event's content. The process to alter,
+append to, or drop Events is called [_filtering_](data-pipeline/filter.md).
 
-There are many use cases when Filtering is required like:
+Use filtering to:
 
-* Append specific information to the Event like an IP address or metadata.
-* Select a specific piece of the Event content.
-* Drop Events that matches certain pattern.
+- Append specific information to the Event like an IP address or metadata.
+- Select a specific piece of the Event content.
+- Drop Events that match a certain pattern.
 
 ## Tag
 
-Every Event that gets into Fluent Bit gets assigned a Tag. This tag is an internal string that is used in a later stage by the Router to decide which Filter or Output phase it must go through.
+Every Event ingested by Fluent Bit is assigned a Tag. This tag is an internal string
```

Comment:
```suggestion
Every Event ingested by Fluent Bit is assigned a tag. This tag is an internal string
```
same thing here about common nouns

Reply:
same thought as before, I don't know if this is common or not.